preload_qsos_dr7.m

```diff
@@ -5,17 +5,7 @@
 % load QSO catalog
 tic;
 
-% ----------dr7----------------------------------------
-variables_to_load = {'all_plate_dr7', 'all_mjd_dr7', 'all_fiber_dr7',...
-'all_QSO_ID','all_zqso'};
-
-
-
-load(sprintf('%s/catalog', processed_directory(release)), ...
-    variables_to_load{:});
-
-num_quasars = numel(all_zqso);
-
+num_quasars = numel(all_zqso_dr7);
```

Comment:
`all_zqso_dr`, appears without a reference? Should we load the variables saved in build_catalog_dr7 first?

Reply:
It is referenced when you run run_script_hpcc_dr7.m, because you've run build_catalog_dr7.m b/f